src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -11056,6 +11062,8 @@ private bool ScanExplicitlyTypedLambda(Precedence precedence)
                     // Advance past the open paren or comma.
                     this.EatToken();
 
+                    _ = ParseAttributeDeclarations(forLambda: true);
```

Comment:
Do we need to do this before the open paren as well?

Reply:
I believe a test for a `static` lambda with a leading attribute will fail without this.

---
In reply to: [599119307](https://github.com/dotnet/roslyn/pull/52038#discussion_r599119307) [](ancestors = 599119307)

Reply:
> Do we need to do this before the open paren as well?

The caller handles the open parenthesis token.

---
In reply to: [599120138](https://github.com/dotnet/roslyn/pull/52038#discussion_r599120138) [](ancestors = 599120138,599119307)